kernel/arch/riscv64/Interrupts.cpp

```diff
@@ -39,73 +40,93 @@ extern "C"
         "ecall from U-mode",
         "ecall from S-mode"
     };
+    constexpr size_t ExceptionStrCount = sizeof(ExceptionStrs) / sizeof(const char*);
+    constexpr uintptr_t InterruptBitMask = 1ul << 63;
+
+    void HandleException(Npk::TrapFrame* frame)
+    {
+        using namespace Npk;
+        using namespace Npk::Memory;
+
+        Memory::VmFaultFlags faultFlags {};
+        switch (frame->vector)
+        {
+        case 12:
```

Comment:
ok here i'm not sure what is the better thing to do, i usually like to see a label instead of a value, or maybe better something like 
```c
case LABEL //Val 
```

so we also know what is the expected value by looking at the statement, but in the same time, i see that is basically setting the action inside the case, so not sure if it will apply or not here. .

So is just something i noticed, and not sure which way is better, so up to you to decide what to do! :)